README.md

```diff
@@ -172,9 +170,21 @@ Application | GitHub Link | Store/Marketplace
 Visual Studio Code | [warpnet/vscode-salt-lint](https://github.com/warpnet/vscode-salt-lint) | [VisualStudio Marketplace](https://marketplace.visualstudio.com/items?itemName=warpnet.salt-lint)
 Sublime Text | [warpnet/SublimeLinter-salt-lint](https://github.com/warpnet/SublimeLinter-salt-lint) | [Package Control](https://packagecontrol.io/packages/SublimeLinter-contrib-salt-lint)
 
-
 Wish to create a `salt-lint` extension for your favourite editor? We're always looking for [contributions](CONTRIBUTING.md)!
 
+# Fix common issues
+
+`sed` might be one of the better tools to fix common issues.
+
+Fix spacing arround `{{ var_name }}`, eg. `{{env}}` --> `{{ env }}`:\
+`sed -i -E "s/\{\{\s?([^}]*[^} ])\s?\}\}/\{\{ \1 \}\}/g"  $(find salt -name '*.sls')`
```

Comment:
Don't assume everyone's salt repo is called `salt`, instead assume they are inside their repository.

```suggestion
The following commands have to be run from within your salt directory/repository:

`sed -i -E "s/\{\{\s?([^}]*[^} ])\s?\}\}/\{\{ \1 \}\}/g"  $(find . -name '*.sls')`
```

Reply:
Yeah, that’s what I thought somewhat later after the commit.
Good update!

Reply:
Since @roaldnefs had already merged, I created a separate pull request (#109) which has just been merged.

Thanks for your contributions @hbokh!